wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -100,11 +101,11 @@ getWAddress cachedAccModifier cAddr = do
     isChange <- getFlag ChangeAddr camChange
     return $ CAddress aId (mkCCoin balance) isUsed isChange
 
-getAccount :: MonadWalletWebMode m => CachedCAccModifier -> AccountId -> m CAccount
-getAccount accMod accId = do
+getAccountMod :: MonadWalletWebMode m => (WalBalancesAndUtxo, CachedCAccModifier) -> AccountId -> m CAccount
```

Comment:
I don't think we benefit from this being a pair, we can use normal curried style.